webui/react/src/e2e/models/pages/SignIn.ts

```diff
@@ -0,0 +1,17 @@
+import { BasePage } from 'e2e/models/BasePage';
+import { DeterminedAuth } from 'e2e/models/components/DeterminedAuth';
+import { PageComponent } from 'e2e/models/components/Page';
+
+/**
+ * Returns a representation of the SignIn page.
+ *
+ * @remarks
+ * This constructor represents the contents in src/pages/SignIn.tsx.
+ *
+ * @param {Page} page - The '@playwright/test' Page being used by a test
+ */
+export class SignIn extends BasePage {
+  readonly pageComponent: PageComponent = new PageComponent({parent: this});
+  readonly detAuth: DeterminedAuth = new DeterminedAuth({parent: this.pageComponent})
+  // TODO add SSO page model as well
```

Comment:
This seems probably out of scope, so feel free to make a ticket for sso. I think a lot more setup has to go into it.